### ISSUE TITLE
install-sysroot.py --running-as-hook' returned non-zero exit status 1

### ISSUE THREAD

**Issue by ignisf** [cowboyd/libv8#237]
```
ignisf@scw-7740a5:~/libv8$ MAKEFLAGS+=-j4 bundle exec rake clean binary
git submodule --quiet foreach git reset --hard
HEAD is now at af0fd4e Update to VS packaging script for dbghelp.dll bug
git submodule --quiet foreach git clean -dxf
git submodule update --init
rm -rf pkg
rm -rf vendor/v8
git clean -dxf -e .bundle -e vendor/bundle
Removing Gemfile.lock
Removing Makefile
Removing vendor/.gclient
ruby ext/libv8/extconf.rb
creating Makefile
Running: gclient root
Running: gclient config --spec 'solutions = [
  {
    "url": "https://chromium.googlesource.com/v8/v8.git",
    "managed": False,
    "name": "v8",
    "deps_file": "DEPS",
    "custom_deps": {},
  },
]
'
Running: gclient sync --with_branch_heads
Error: Command '/usr/bin/python v8/build/linux/sysroot_scripts/install-sysroot.py --running-as-hook' returned non-zero exit status 1 in /home/ignisf/libv8/vendor
Traceback (most recent call last):
  File "/home/ignisf/libv8/vendor/depot_tools/fetch.py", line 300, in <module>
    sys.exit(main())
  File "/home/ignisf/libv8/vendor/depot_tools/fetch.py", line 295, in main
    return run(options, spec, root)
  File "/home/ignisf/libv8/vendor/depot_tools/fetch.py", line 289, in run
    return checkout.init()
  File "/home/ignisf/libv8/vendor/depot_tools/fetch.py", line 132, in init
    self.run_gclient(*sync_cmd)
  File "/home/ignisf/libv8/vendor/depot_tools/fetch.py", line 76, in run_gclient
    return self.run(cmd_prefix + cmd, **kwargs)
  File "/home/ignisf/libv8/vendor/depot_tools/fetch.py", line 66, in run
    return subprocess.check_output(cmd, **kwargs)
  File "/usr/lib/python2.7/subprocess.py", line 574, in check_output
    raise CalledProcessError(retcode, cmd, output=output)
subprocess.CalledProcessError: Command '('gclient', 'sync', '--with_branch_heads')' returned non-zero exit status 2
/home/ignisf/libv8/ext/libv8/builder.rb:113:in `block in setup_build_deps!': unable to fetch v8 source (RuntimeError)
	from /home/ignisf/libv8/ext/libv8/builder.rb:111:in `chdir'
	from /home/ignisf/libv8/ext/libv8/builder.rb:111:in `setup_build_deps!'
	from /home/ignisf/libv8/ext/libv8/builder.rb:67:in `build_libv8!'
	from /home/ignisf/libv8/ext/libv8/location.rb:24:in `install!'
	from ext/libv8/extconf.rb:7:in `<main>'
rake aborted!
Command failed with status (1): [ruby ext/libv8/extconf.rb...]
/home/ignisf/libv8/Rakefile:22:in `block in <top (required)>'
/home/ignisf/vendor/ruby/2.3.0/gems/rake-11.3.0/exe/rake:27:in `<top (required)>'
Tasks: TOP => binary => compile
(See full trace by running task with --trace)
ignisf@scw-7740a5:~/libv8$ cd vendor/
ignisf@scw-7740a5:~/libv8/vendor$ /usr/bin/python v8/build/linux/sysroot_scripts/install-sysroot.py --running-as-hook
Unrecognized host arch: aarch64
ignisf@scw-7740a5:~/libv8/vendor$
```

**Comment by ignisf** [comment]
An upstream issue exists: https://bugs.chromium.org/p/v8/issues/detail?id=5075

**Comment by ignisf** [comment]
This will be fixed with a stable upstream 6.0 release.
